Compiled from "FaultHandling.scala"
public final class akka.actor.dungeon.FaultHandling$$anonfun$2 extends scala.runtime.AbstractFunction1<java.lang.Throwable, scala.runtime.BoxedUnit> implements scala.Serializable {
  public static final long serialVersionUID;

  private final akka.actor.ActorCell $outer;

  public final void apply(java.lang.Throwable);
    Code:
       0: aload_0
       1: getfield      #20                 // Field $outer:Lakka/actor/ActorCell;
       4: getstatic     #26                 // Field scala/collection/immutable/Nil$.MODULE$:Lscala/collection/immutable/Nil$;
       7: aload_1
       8: invokevirtual #32                 // Method akka/actor/ActorCell.handleInvokeFailure:(Lscala/collection/immutable/Iterable;Ljava/lang/Throwable;)V
      11: return

  public final java.lang.Object apply(java.lang.Object);
    Code:
       0: aload_0
       1: aload_1
       2: checkcast     #39                 // class java/lang/Throwable
       5: invokevirtual #41                 // Method apply:(Ljava/lang/Throwable;)V
       8: getstatic     #47                 // Field scala/runtime/BoxedUnit.UNIT:Lscala/runtime/BoxedUnit;
      11: areturn

  public akka.actor.dungeon.FaultHandling$$anonfun$2(akka.actor.ActorCell);
    Code:
       0: aload_1
       1: ifnonnull     6
       4: aconst_null
       5: athrow
       6: aload_0
       7: aload_1
       8: putfield      #20                 // Field $outer:Lakka/actor/ActorCell;
      11: aload_0
      12: invokespecial #54                 // Method scala/runtime/AbstractFunction1."<init>":()V
      15: return
}
